### PR TITLE
Fix packaging and exports all modules for Typescript/Javascript

### LINKED ISSUE #26: Use `cosmian_js_lib` as dependency in another project (`cloudproof_demos` for example)
Check if everything is working (CoverCrypt, Findex)

### LINKED ISSUE #32: Add README.md

### PR DESCRIPTION
**Pull request by Manuthor**
---
### Added
- Add README.md
### Changed
- Re-enable ABE demo via abe.html
### Fixed
- Fix packaging and exports all modules for Typescript/Javascript
- [Findex tests/PostgREST] Increase again delay after users provisioning
### Removed

Closes #32 
Closes #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Cosmian Javascript Lib
+
+The library provides a Typescript friendly API to the [Cosmian Ubiquitous Encryption platform](https://cosmian.com),
+the CloudProof product in particular.
```
Comment (ThibaudDauce) [inline README.md:4]: > The library provides a Typescript friendly API to the Cosmian Ubiquitous Encryption platform the CloudProof product in particular.

It seems there is a problem in this sentence :-)

File: site/index.html [hunk 2]
```diff
@@ -82,7 +82,7 @@ <h2>Cloudproof Encryption</h2>
     <div class="menu">
         <button onclick="location.href='findex.html';">FINDEX</button>
         <button onclick="location.href='cloudproof.html';">CLOUDPROOF ENCRYPTION</button>
-        <!-- <button onclick="location.href='abe.html';">ABE DEMO</button> -->
+        <button onclick="location.href='abe.html';">ATTRIBUTE BASED ENCRYPTION</button>
```
Comment (ThibaudDauce) [inline site/index.html:85]: Why not using link here ?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+This library free software and is available on NPM public repository
+
+```bash
+npm i cosmian_js_lib
+```
+
+## Versions Correspondence
+
+Local encryption and decryption with [GPSW](https://github.com/Cosmian/abe_gpsw) or [CoverCrypt](https://github.com/Cosmian/cover_crypt) and SSE Findex Cosmian scheme use WASM librairies which is transparent for Javascript/Typescript usage.
```
Comment (tbrezot) [inline README.md:31]: I am not sure of this one:
```suggestion
Local encryption and decryption with [GPSW](https://github.com/Cosmian/abe_gpsw) or [CoverCrypt](https://github.com/Cosmian/cover_crypt) and SSE Findex Cosmian scheme use WASM librairies which are transparent for Javascript/Typescript usage.
```

**Review by tbrezot — APPROVED** [review]

File: site/index.html (hunk 2 above)
Comment (Manuthor) [reply site/index.html:85]: there is a CSS over `button`, that is the main reason imho

**Pull request merged**